mmv1/templates/terraform/decoders/vertex_ai_models.go.tmpl

```diff
@@ -0,0 +1,7 @@
+if modelName, ok := res["model"].(string); ok && modelName != "" {
+	res["name"] = modelName
+}
+
+res["modelName"] = res["displayName"]
```

Comment:
As the schema on this PR contain's the API's displayName field (https://github.com/GoogleCloudPlatform/magic-modules/pull/12074/commits/f1124737c6c9bba113958949f79f51ee608a90d0) can we remove this decoder that translates the API's displayName field into the made up modelName field?

```suggestion
```

Reply:
yes this should've been removed. This is meant for the upload method PR